db/db_impl/db_impl.h

```diff
@@ -954,6 +954,11 @@ class DBImpl : public DB {
 
   VersionSet* GetVersionSet() const { return versions_.get(); }
 
+  // Wait for any compaction
+  // We add a bool parameter to wait for unscheduledCompactions_ == 0, but this
+  // is only for the special test of CancelledCompactions
+  Status WaitForCompact(bool waitUnscheduled = false);
```

Comment:
There is a `TEST_WaitForCompact`, can it be used here?

Reply:
it's the same function, but we want to move it out of DEBUG block, so we can run it in release build. And we do want to run benchmark code in release build instead of debug build.